wdio.conf.js

```diff
@@ -0,0 +1,366 @@
+  //
+  capabilities: {
+    browserSpock: {
+      desiredCapabilities: {
+        browserName: `firefox`,
+        tunnelIdentifier: process.env.SC_TUNNEL_IDENTIFIER
+      }
+    },
+    browserMccoy: {
+      desiredCapabilities: {
+        browserName: `chrome`,
+        chromeOptions: {
+          args: [
+            `--use-fake-device-for-media-stream`,
+            `--use-fake-ui-for-media-stream`
+          ]
+        },
+        tunnelIdentifier: process.env.SC_TUNNEL_IDENTIFIER
+      }
+    }
+  },
+  //
+  // ===================
+  // Test Configurations
+  // ===================
+  // Define all options that are relevant for the WebdriverIO instance here
+  //
+  // By default WebdriverIO commands are executed in a synchronous way using
+  // the wdio-sync package. If you still want to run your tests in an async way
+  // e.g. using promises you can set the sync option to false.
+  sync: true,
+  //
+  // Level of logging verbosity: silent | verbose | command | data | result | error
+  logLevel: `error`,
+  //
+  // Enables colors for log output.
+  coloredLogs: true,
+  //
+  // If you only want to run your tests until a specific amount of tests have failed use
+  // bail (default is 0 - don't bail, run all tests).
+  bail: 0,
+  //
+  // Saves a screenshot to a given path if a command fails.
+  screenshotPath: `./reports/screenshots/`,
+  //
+  // Set a base URL in order to shorten url command calls. If your url parameter starts
+  // with "/", then the base url gets prepended.
+  baseUrl: `http://localhost:${PORT}/packages/node_modules`,
+  //
+  // Default timeout for all waitFor* commands.
+  waitforTimeout: 10000,
+  //
+  // Default timeout in milliseconds for request
+  // if Selenium Grid doesn't send response
+  connectionRetryTimeout: 90000,
+  //
+  // Default request retries count
+  connectionRetryCount: 3,
+  //
+  // Initialize the browser instance with a WebdriverIO plugin. The object should have the
+  // plugin name as key and the desired plugin options as properties. Make sure you have
+  // the plugin installed before running any tests. The following plugins are currently
+  // available:
+  // WebdriverCSS: https://github.com/webdriverio/webdrivercss
+  // WebdriverRTC: https://github.com/webdriverio/webdriverrtc
+  // Browserevent: https://github.com/webdriverio/browserevent
+  // plugins: {
+  //     webdrivercss: {
+  //         screenshotRoot: 'my-shots',
+  //         failedComparisonsRoot: 'diffs',
+  //         misMatchTolerance: 0.05,
+  //         screenWidth: [320,480,640,1024]
+  //     },
+  //     webdriverrtc: {},
+  //     browserevent: {}
+  // },
+  //
+  // Test runner services
+  // Services take over a specific job you don't want to take care of. They enhance
+  // your test setup with almost no effort. Unlike plugins, they don't add new
+  // commands. Instead, they hook themselves up into the test process.
+  services: CI ? [
+    `sauce`,
+    `static-server`,
+    `webpack`
```

Comment:
Ha, the last time I tried this, they didn't even support webpack 2 configs. Good to see that they are working now.